cico_setup.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# functions to generate latest client if any change in design pkg
+source <(curl -ssL https://git.io/fxP0E)
```

Comment:
Why using your own personal repo?

Reply:
no specific reason. I have moved it in fabric8-common. See #https://github.com/fabric8-services/fabric8-common/pull/36